packages/music/src/classes/Queue.ts

```diff
@@ -632,4 +632,50 @@ export abstract class Queue<T extends Player = Player> {
     this.processQueue();
     return tracks;
   }
+  /**
+   * Play apple
+   * @param search
+   * @param options
+   * @param enqueueTop
+   * @returns
+   */
+   public async apple(
+    search: string,
+    options?: ITrackOptions,
+    enqueueTop?: boolean
+  ): Promise<YoutubeTrack[] | undefined> {
+    const appleTracks =
+      typeof search === "string" ? await Util.getAppleTracks(search) : search;
```

Comment:
this is unnecessary check since search is always string

Reply:
i was just copying what the spotify one had figured you must have a reason for it